features/step_definitions/_then_steps.rb

```diff
@@ -89,9 +89,9 @@
 end
 
 Then /^show me the list of stories$/ do
-  stories = Story.find(:all, conditions: "project_id=#{@project.id}", order: 'position ASC')
+  stories = Story.where(project_id: @project.id).order('position ASC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.